### PR TITLE
Bug/performance layer list

### PR DESCRIPTION
**Pull request by mtoonen**
Fix for slow ajaxrequests in Gouda. This moves that functionality to the front-end.

### CODE REVIEW

File: viewer-config-persistence/src/main/java/nl/b3p/viewer/util/databaseupdate/DatabaseSynchronizer.java [hunk 1]
```diff
@@ -174,6 +174,11 @@
         
         // 5.7.3
         updates.put("39", new UpdateElement(Collections.singletonList("add_geoservice_geofence_header.sql"), String.class, true));
+
+        //5.6.4
+
+        updates.put("40", new UpdateElement (Collections.singletonList("selectedcontentcaches_dirty.sql"), String.class));
```
Comment (mprins) [inline viewer-config-persistence/src/main/java/nl/b3p/viewer/util/databaseupdate/DatabaseSynchronizer.java:180]: this file is missing from this PR

**Review by mprins — CHANGES_REQUESTED** [review]
seems therr is a file missing. 
Also what are expected problems on downstream projects? eg oigs, ibis, ... that have custom components. a quick note would be good on API changes

File: viewer-config-persistence/src/main/java/nl/b3p/viewer/util/databaseupdate/DatabaseSynchronizer.java (hunk 1 above)
Comment (mtoonen) [reply viewer-config-persistence/src/main/java/nl/b3p/viewer/util/databaseupdate/DatabaseSynchronizer.java:180]: It's an older, already existing file which I reused

**Comment by mtoonen** [comment]
> seems therr is a file missing.
> Also what are expected problems on downstream projects? eg oigs, ibis, ... that have custom components. a quick note would be good on API changes

Not really an API change: backend didn't change (thought about removing the LayerListActionBean, but decided against it for these reason). Everything stayed the same, except for no ajaxcalls for retrieving the layerlist

**Review by mprins — APPROVED** [review]

**Pull request merged**